js/function.js

```diff
@@ -0,0 +1,53 @@
+// Функция для проверки длины строки
+
+const searchStringValidation = (test, number) => {
```

Comment:
У параметров должны быть понятные имена, number это слишком общее, а test кажется что еще более общее
```suggestion
const searchStringValidation = (string, maxLength) => {
```